packages/babel-runtime-corejs3/helpers/esm/AsyncGenerator.js

```diff
@@ -2,7 +2,7 @@ import _Promise from "core-js-pure/features/promise/index.js";
 import _Symbol from "core-js-pure/features/symbol/index.js";
 import _Symbol$asyncIterator from "core-js-pure/features/symbol/async-iterator.js";
 import OverloadYield from "./OverloadYield.js";
-function AsyncGenerator(e) {
+function AsyncGeneratorImpl(e) {
```

Comment:
Can you re-run `make build` to regenerate these?

Reply:
Removed in [`8ebaf2e` (#16510)](https://github.com/babel/babel/pull/16510/commits/8ebaf2edc93a536e5fe776c0b71462300c19c53c).